### PR TITLE
Finished advanced config in README.md

### PR DESCRIPTION
**Pull request by BartKoppelmans**
Also added name to contributors list

### CODE REVIEW

**Comment by joshho** [comment]
The description for "initial_transfer" is not completely correct. It will transfer newly caught pokemon even if you have initial_transfer set to 0.

File: README.md [hunk 1]
```diff
@@ -174,16 +174,18 @@ To update your project do: `git pull` in the project folder
...
+------ | -------
+`max_steps` |		The steps around your initial location (DEFAULT 5 mean 25 cells around your location) that will be explored
+`mode` |  		Set farming Mode for the bot ('all', 'poke', 'farm'). 'all' means both spinning pokéstops and catching pokémon; 'poke'means only catching pokémon and 'farm' means only spinning pokéstops
+`walk` | Walk with the given speed (meters per second max 4.16 because of walking end on 15km/h)
+`debug` | 		Let the default value here except if you are developer
+`test` | 		Let the default value here except if you are developer
+`initial_transfer` | 	Set this to 1 if you want to transfer pokemon
+`location_cache` | 	Bot will start at last known location
+`distance_unit` | 	Set the unit to display distance in (e.g, km for kilometers, mi for miles, ft for feet)
+`item_filter` | 	Pass a list of unwanted items to recycle when collected at a Pokestop (e.g, [\"101\",\"102\",\"103\",\"104\"] to recycle potions when collected)
```
Comment (tdurieux) [inline README.md]: the "\" are not required

**Comment by BartKoppelmans** [comment]
Better like this?

**Comment by VictorChen** [comment]
Is the `initial_transfer` description correct? I believe it will transfer pokemons with CP less than what the value of `initial_transfer` is.

Ex:
```
initial_transfer: 1000
```
Will transfer all duplicate pokemons with CP less than 1000

Ex:
```
initial_transfer: 0
```
Will not transfer pokemons

Correct me if I'm wrong...

**Comment by BartKoppelmans** [comment]
Another commit containing a new description has been made!

**Comment by SteffanLong** [comment]
Can you fix the item option? it's CSV now, not an array. It should look like `"101,102,103,104"`. I see no reason not to merge after that is fixed. :)

**Comment by BartKoppelmans** [comment]
Done!

**Comment by Calcyfer** [comment]
For `location_cache` , this would clarify a few things ;

`location_cache` | Bot will start at last known location if you do not have `location` set in the config

**Comment by BartKoppelmans** [comment]
Fixed that! @Calcyfer

**Comment by Calcyfer** [comment]
And fast. Descriptions seem great to me!

**Comment by BartKoppelmans** [comment]
Okay cool! Please let me know if I can do anything else, either for this PR or for any other thing!

**Comment by solderzzc** [comment]
GITHUB: This branch has conflicts that must be resolved

**Comment by solderzzc** [comment]
please reopen to notify us :)

**Pull request closed without merging**